core/shared/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -38,6 +37,8 @@ import org.typelevel.vault._
 import java.io._
 import java.net.URL
 
+import fs2.io._
```

Comment:
group this with the other fs2.io imports then the scalafix warning should go away

Reply:
You can also run `scalafixAll` and it will fix it for you :)

Reply:
I thought lintQuick would do that no?